### PR TITLE
Add support for configuring system timezone

### PR DESCRIPTION
**Pull request by erhudy**

### CODE REVIEW

File: cookbooks/bcpc/recipes/system.rb [hunk 1]
```diff
@@ -124,3 +124,9 @@
     devices_to_converge.length.zero?
   end
 end
+
+# set timezone (note that to ensure this takes full effect, you should reboot afterward)
+bash "set timezone to #{node['bcpc']['time_zone']}" do
+  code "timedatectl set-timezone #{node['bcpc']['time_zone']}"
+  not_if { %x[timedatectl status | grep Timezone | awk '{ print $2 }'].strip == node['bcpc']['time_zone'] }
```
Comment (kamidzi) [inline cookbooks/bcpc/recipes/system.rb]: minor point, but it's not really necessary to call grep before awk:

`awk '/Timezone/{print $2}'`

**Comment by kamidzi** [comment]
Locally this seems to check out.

**Comment by erhudy** [comment]
Rebased and squashed commits.

**Comment by erhudy** [comment]
Withdrawing this PR - will redo it later when we have the opportunity to build new clusters with UTC as the clock setting from the outset.

**Pull request closed without merging**